services/checks/checks.go

```diff
@@ -1422,40 +1407,12 @@ func (s *Service) downloadChecks(ctx context.Context) ([]check.Check, error) {
 		return nil, nil
 	}
 
-	s.l.Infof("Downloading checks from %s ...", s.host)
-
-	nCtx, cancel := context.WithTimeout(ctx, platformRequestTimeout)
+	nCtx, cancel := context.WithTimeout(ctx, portalRequestTimeout)
```

Comment:
nit: naming of `nCtx` is not clear, what about `portalRequestCtx`?

Reply:
This a common naming in PMM projects, also scope of this variable is just few lines. I'm not sure that long var name will bring any clarity here.

Reply:
what does `n` stands for? what if we have more contexts? it will become n2Ctx?

Reply:
Just `new context`. If you have several contexts of course you should use more meaningful names. But when your variable lives just few lines of code better to use short variable names.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names